src/layouts/Course_Offering__c-HEDA Course Offering Layout.layout

```diff
@@ -72,6 +76,10 @@
         <layoutColumns/>
         <style>CustomLinks</style>
     </layoutSections>
+    <relatedLists>
```

Comment:
Should there be more fields defined here? I'm actually having trouble figuring out what related list this is - Course Offering Periods, I think?

Reply:
@lifewithryan Not sure if you saw this, forgot to tag you

Reply:
Ahhh lemme look.